syntax = "proto3";
import "PacketsId.proto";

package my.proto.package;

message MousePositionMessage
{
	option (package_id) = MOUSE_POSITION_MESSAGE;
	int32 x = 1;
	int32 y = 2;
}
